tests/test_server.py

```diff
@@ -57,3 +62,17 @@ def test_metadata(connected_server):
   assert n_pes == 2
   assert n_nodes == 2
   assert node_size == 1
+
+def test_receive_response_msg(connected_server):
```

Comment:
Does this need a corresponding Charm++ server code? Could you add that as well to the PR

Reply:
yes, most of the tests in the file actually need the server to be running.